jamal-snippet/src/main/java/javax0/jamal/snippet/Sort.java

```diff
@@ -0,0 +1,134 @@
+package javax0.jamal.snippet;
+
+import javax0.jamal.api.*;
+import javax0.jamal.tools.InputHandler;
+import javax0.jamal.tools.Params;
+import javax0.jamal.tools.Scan;
+
+import java.text.Collator;
+import java.util.*;
+import java.util.function.Function;
+import java.util.regex.Pattern;
+import java.util.stream.Stream;
+
+import static java.util.stream.Collectors.toList;
+import static javax0.jamal.tools.InputHandler.skipWhiteSpaces;
+import static javax0.jamal.tools.Params.holder;
+
+public class Sort implements Macro, InnerScopeDependent {
+
+	@Override
+	public String evaluate(Input in, Processor processor) throws BadSyntax {
+		final var separator = holder("separator").orElse("\n").asPattern();
+		final var join = holder("join").orElse("\n").asString();
+		final var locale = holder("locale").asString();
+		final var columns = holder("columns").asString();
+		final var pattern = holder("pattern").asPattern();
+		final var numeric = holder("numeric").asBoolean();
+		final var reverse = holder("reverse").asBoolean();
```

Comment:
The comment that I made for `join` holds for the other parameters as well.

If we want some macro names, then it has to be something like `sort$join`, `sort$columns` etc.